userspace/libsinsp/eventformatter.cpp

```diff
@@ -106,10 +96,10 @@ void sinsp_evt_formatter::set_format(output_format of, const std::string& fmt)
 
 			if(last_nontoken_str_start != j)
 			{
-				rawstring_check* newtkn = new rawstring_check(lfmt.substr(last_nontoken_str_start, j - last_nontoken_str_start));
-				m_tokens.emplace_back(std::make_pair("", newtkn));
+				auto newtkn = std::make_unique<rawstring_check>(lfmt.substr(last_nontoken_str_start, j - last_nontoken_str_start));
+				m_tokens.emplace_back(std::make_pair("", newtkn.get()));
 				m_tokenlens.push_back(0);
-				m_chks_to_free.push_back(newtkn);
+				m_chks_to_free.push_back(std::move(newtkn));
```

Comment:
Do we still need `m_chks_to_free` here? The `~sinsp_evt_formatter()` dtor was deleted.

Reply:
These definitely get destroyed by `~sinsp_evt_formatter()`. I think the name is just misleading, I propose to rename this `m_checks` to make it explicit.

Reply:
Oh i see, sorry i completely misread the code :D